Source/Scene/Model.js

```diff
@@ -448,6 +455,27 @@ define([
         this._id = options.id;
 
         /**
+         * Returns the height reference of the model
+         *
+         * @memberof Model.prototype
+         *
+         * @type {HeightReference}
+         *
+         * @default HeightReference.NONE
+         */
+        this.heightReference = defaultValue(options.heightReference, HeightReference.NONE);
+        this._heightReference = this.heightReference;
+        this._heightChanged = false;
+        this._removeUpdateHeightCallback = undefined;
+        var scene = options.scene;
+        if (defined(scene)) {
+            this._scene = scene;
```

Comment:
Set this above, outside of the `if` statement, so we don't run into any issues with browser's not creating a hidden class.